recipe/meta.yaml

```diff
@@ -112,25 +146,11 @@ outputs:
     script: build-aspnetcore.bat  # [win]
     script: build-aspnetcore.sh  # [not win]
     requirements:
-      build:
-        - {{ stdlib('c') }}
-        - {{ compiler('cxx') }}
-        - {{ stdlib('c') }}
-        - posix                  # [win]
-      host:
-        - zlib                   # [unix]
-        - libcurl                # [unix]
-        - lttng-ust              # [linux64]
       run:
-        - lttng-ust                 # [linux64]
-        - libcurl                   # [unix]
-        - icu                       # [unix]
-        - krb5                      # [unix]
-        - __glibc >=2.17            # [linux64]
         - dotnet-runtime ={{ runtime_version }}
```

Comment:
Out of curiosity, why doesn't this section need the
```
run_exports:
        - {{ pin_subpackage("dotnet-runtime", max_pin="x.x") }}
``` 
as above?

Reply:
Maybe this syntax is safer:
`dotnet-runtime ={{ runtime_version }}`
the runtime_version seems to change with each sdk_version